src/figuresClasses.ts

```diff
@@ -1,19 +1,102 @@
 export interface Figure {
+  color: 'red' | 'green' | 'blue';
+  shape: 'triangle' | 'rectangle' | 'circle';
+  getArea(): number;
+  a?: number;
+  b?: number;
+  c?: number;
+  radius?: number;
+  width?: number;
+  height?: number;
+}
 
+function getRound(num: number): number {
+  return Math.trunc(num * 100) / 100;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  constructor(
+    public color: 'red' | 'green' | 'blue',
```

Comment:
add an enum and use it here
```suggestion
    public color: Color,
```